webapp/src/ts/services/format-date.service.ts

```diff
@@ -107,14 +107,18 @@ export class FormatDateService {
     const today = moment().startOf('day');
     const diff = date.diff(today, 'days');
 
-    if (diff <= 0) {
+    if (diff < 0) {
       if (this.config.taskDaysOverdue) {
         return this.translateService.instant('task.overdue.days', { DAYS: Math.abs(diff) });
       }
 
       return this.translateService.instant('task.overdue');
     }
 
+    if (diff === 0) {
+      return this.translateService.instant('Due today');
```

Comment:
Please follow the conventions for adding new translation fields here. 
Please use a dot separated name (like `task.due.today`) and add a translation for it in all the translation files: https://github.com/medic/cht-core/tree/master/api/resources/translations

Because we don't want to change behavior for existent deployments when adding this new translation field, when adding the translation please make it the same values as the one for `task.overdue`